andela/hc-rogue-one
# Ft  156577848 allocate priority to check

#### What does this PR do?
User should be able to allocate different priorities to different health checks as per a pre-defined priority criteria
#### Description of Tasks to be complete
1. Assign Priority of a check upon check creation.
2. Create section for setting check priority on dashboard
3. Set priority criteria
4. Update check priority on the dashboard

#### How should this be manually tested?
1: Install the project and install the requirements.txt
2. Make the migrations ` python manage.py makemigrations`
3. Migrate `python manage.py migrate`
2: Start the server by running ` python manage.py runserver `, open localhost  http://localhost:8080 in the browser
3: The landing page/home page is opened
4: You should be able to see the neccessary information displayed on the landing page.
#### What are the relevant pivotal tracker stories?
#156577848
#### Checklist: I completed these to help reviewers

No? Unsure? -> configure your IDE, format the code and add the changes with git add . && git commit --amend
existing code that was well tested you do not have to add tests)

No? -> write tests and add them to this commit git add . && git commit --amend


No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.


No? Unsure? -> execute command git pull --rebase origin develop
